bin/console

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env ruby
 
+require 'rails/all'
```

Comment:
This feels like it should not be necessary

Reply:
I thought the same. Wasn't able to use the 'gem' console without it. Maybe there is another env or require at a higher level I'm missing.